src/test/java/com/springbootassignment/cruddemo/entity/CourseTest.java

```diff
@@ -0,0 +1,115 @@
+package com.springbootassignment.cruddemo.entity;
+
+import com.springbootassignment.cruddemo.entity.Course;
+import com.springbootassignment.cruddemo.entity.Student;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+ class CourseTest {
```

Comment:
Testing for entities, repositories isn't required.
Controller and Service layer testing is enough.